.travis.yml

```diff
@@ -14,6 +14,11 @@ install: travis_retry composer install --prefer-source
 
 script: composer ci
 
+after_success:
+  - vendor/bin/phpunit --coverage-clover coverage.clover
```

Comment:
I tried to find documentation for `after_success` but find it insufficient. I *believe* this is not run for every PHP version but only once after all PHP versions are tested.

I believe this still adds an extra run of PHPUnit on top of the six for the six different PHP versions. Is this correct? I believe this is ok. I believe creating coverage reports six times (and only using one) would be much slower than six runs without and a sevenths with coverage.

One question remains: which PHP version is used for this sevenths run?